esp32c2-hal/Cargo.toml

```diff
@@ -44,15 +44,15 @@ ssd1306            = "0.8.4"
 portable-atomic    = "1.5.1"
 static_cell        = { version = "2.0.0", features = ["nightly"] }
 esp-hal-common     = { features = ["portable-atomic"], path = "../esp-hal-common" } # enable portable-atomic by default for examples
-hex-literal       = "0.4.1"
-crypto-bigint     = {version = "0.5.3", default-features = false }
-elliptic-curve    = {version = "0.13.6", default-features = false, features = ["sec1"] }
-p192              = {version = "0.13.0", default-features = false, features = ["arithmetic"] }
-p256              = {version = "0.13.2", default-features = false, features = ["arithmetic"] }
+hex-literal        = "0.4.1"
+crypto-bigint      = { version = "0.5.3", default-features = false }
+elliptic-curve     = { version = "0.13.6", default-features = false, features = ["sec1"] }
+p192               = { version = "0.13.0", default-features = false, features = ["arithmetic"] }
+p256               = { version = "0.13.2", default-features = false, features = ["arithmetic"] }
 embassy-sync       = "0.4.0"
 
 [features]
-default              = ["rt", "vectored", "xtal-40mhz"]
+default              = ["rt", "vectored", "xtal-40mhz", "embassy-integrated-timers"]
```

Comment:
This is slightly annoying to use. If the user needs a generic-queue feature, they have to disable default features then re-add 3 of them. But: the change is backwards compatible at least and speedy-by-default.